src/keyszer/config_api.py

```diff
@@ -41,6 +41,28 @@
 # multipurpose timeout
 _TIMEOUTS = TIMEOUT_DEFAULTS
 
+from .output import set_keystroke_delay
+
+# delay to fix Unicode entry sequence failures
+unicode_delay_ms = 0
+
+
+def throttle_delays(unicode_delay=0, keystroke_delay=0):
+    global unicode_delay_ms
+    _ud = unicode_delay
+    _kd = keystroke_delay
+    if _ud <= 100 and _ud >= 0 and isinstance(_ud, int):
+        unicode_delay_ms = _ud / 1000
```

Comment:
It's no longer "ms" once you divide by 1000, so this is incorrectly named.  I'd try keeping the variables integers unless the last moment (if that is how we expect they will be assigned), and then writing a tiny `sleepms` helper to abstract the `/1000`...

Reply:
Well, technically the original variable is just an integer with no inherent unit, although yes it is intended to be milliseconds. So I'm turning the original unitless integer into actual milliseconds of delay with the normal `sleep`. But yes, this is at least awkward and I've been thinking about simplifying it. 

There is a `usleep` already in `config_api`, I should probably just try to import and use that instead of messing with giving milliseconds to `time.sleep`. Or recreate it, since that was more intended for use in the user's config.

Reply:
The inner function can be pulled out, sure... I think `usleep` as an API helper will be going away... once we get the timing configs tweaks right there shouldn't be a need for commands that sleep - and someone can always add them back manually if they TRULY think they need them.

Reply:
Had to make a new simple wrapper `sleep_ms` in `config_api` and then import it (and also use it in the Unicode function in `config_api`). 

> I think usleep as an API helper will be going away

That sounds fine. The sleep delays I needed in config were more like getting it to wait until a new tab was available to type "about:preferences" into. I don't think `usleep` really needs to be there. It's perfectly easy to do something like `sleep(0.05)`. As long as the main `sleep` helper works, it should handle most config needs.